lib/ansible/utils/module_docs_fragments/shell_common.py

```diff
@@ -32,7 +32,7 @@ class ModuleDocFragment(object):
       - name: ansible_system_tmpdirs
   async_dir:
     description:
-       - Directory in which ansible will keep async job inforamtion
+       - Directory in which ansible will keep async job information
```

Comment:
Should we also document the envvar deprecation here?

Reply:
Well I'm not sure whether we want to deprecate the env var in general, it's really people who do this on a task that is deprecated.

```
environment:
    ANSIBLE_ASYNC_DIR: /path
```